brave/brave-core
# Site banner now transitions to closed after three seconds when a donation has completed

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Start Brave and add funds (if none exist)
2. Visit a site, click BAT icon and contribute a page donation
3. Verify the 'Thank you' banner closes after 3 seconds

## Reviewer Checklist:
